docs/shared_vip.md

```diff
@@ -0,0 +1,100 @@
+
+```
+NAME                      TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)        AGE
+sharedvip-avisvc-lb1      LoadBalancer   10.108.153.227   100.64.196.61   80:31658/TCP   6d23h
+sharedvip-avisvc-lb2      LoadBalancer   10.102.147.29    100.64.196.61   80:31331/UDP   6d23h
+```
+
+In case their is a requirement to set a preferred static VIP through the `.spec.loadBalancerIP` field in the Service, it is required that all LB Services sharing the Annotation value must have the same preferred VIP provided in the spec. If two Services under the same Annotation value have different static VIP set, no Virtual Service will be configured. This is treated as a misconfiguration and will be logged in AKO accordingly.
+
+An example of configuring multiple LB Services to share a preferred VIP is shown below.
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    ako.vmware.com/enable-shared-vip: "shared-vip-key-1"
+  name: sharedvip-avisvc-lb1
+  namespace: default
+spec:
+  type: LoadBalancer
+  loadBalancerIP: 100.64.196.75
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: avi-server
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    ako.vmware.com/enable-shared-vip: "shared-vip-key-1"
+  name: sharedvip-avisvc-lb2
+  namespace: default
+spec:
+  type: LoadBalancer
+  loadBalancerIP: 100.64.196.75
+  ports:
+  - port: 80
+    protocol: UDP
+    targetPort: 8080
+  selector:
+    app: avi-server
+```
+
+The expected status message should have the VIP matching the preferred static IP provided in the Service spec.
+```
+NAME                      TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)        AGE
+sharedvip-avisvc-lb1      LoadBalancer   10.108.153.227   100.64.196.75   80:31658/TCP   6d23h
+sharedvip-avisvc-lb2      LoadBalancer   10.102.147.29    100.64.196.75   80:31331/UDP   6d23h
+```
+
+
+There are a few things that must be considered while configuring the Services with the aforementioned annotation;
+1. Make sure that LoadBalancer Services which are intended to share a VIP, must have the **same** annotation value. As shown in the example shown above, the annotation value `shared-vip-key-1` is same for both Services.
+2. In order to avoid any errors while configuring the Virtual Service on the Avi controller, it is required that there is no conflicting Port-Protocol pairs in the LB Services that share the Annotation value. With the example shown above both Services are exposing a unique, non-conflicting Port-Protocol for the backend application i.e. 80/TCP and 80/UDP. Explicit checks will be added around this to ensure misconfigurations in future AKO releases.
+3. The annotation must be provided only on Service of type LoadBalancers. In case the annotation is not provided on the LoadBalancer Service, a dedicated L4 Virtual Service is created corresponding to the Service.
```

Comment:
I think Second part of 3rd point is existing behaviour. So we should not mention it explicitly